pxc-80/Dockerfile.k8s

```diff
@@ -101,6 +101,20 @@ RUN mkdir -p /etc/mysql/conf.d/ /var/log/mysql /var/lib/mysql /docker-entrypoint
 	&& chown -R 1001:1001 /etc/mysql/ /var/log/mysql /var/lib/mysql /docker-entrypoint-initdb.d /etc/percona-xtradb-cluster.conf.d \
 	&& chmod -R g=u /etc/mysql/ /var/log/mysql /var/lib/mysql /docker-entrypoint-initdb.d /etc/percona-xtradb-cluster.conf.d
 
+ARG DEBUG
+RUN [ -n "$DEBUG" ] \
+        && sed -i '/\[mysqld\]/a wsrep_log_conflicts\nlog_error_verbosity=3\nwsrep_debug=1' /etc/mysql/node.cnf \
+        && sed -i 's/exec "$@"/"$@" || sleep infinity/' /entrypoint.sh \
```

Comment:
There's potential for crashing pod if it finds wsrep_start_position:

```
+ sleep infinity --wsrep_start_position=f0b6fcbd-7964-11ea-9d3a-9f767bafc016:346457
sleep: invalid option -- '-'
```

Reply:
it is already fixed in master branch